pkg/build/api/types.go

```diff
@@ -30,6 +30,9 @@ const (
 	// DropCapabilities is an environment variable that contains a list of capabilities to drop when
 	// executing a Source build
 	DropCapabilities = "DROP_CAPS"
+	// JenkinsDefaultServiceName is the default name of the Jenkins service deployed by
+	// JenkinsPipelineBuildStrategy.
+	JenkinsDefaultServiceName = "ci"
```

Comment:
openshift-jenkins? managed-jenkins? Something that differentiates from a potential user Jenkins would be good, although impossible to enforce.